projects/notary/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/notaryproject/notary"
+main_repo: "https://github.com/notaryproject/notary"
+primary_contact: ""
```

Comment:
For the primary_contact and auto_ccs, is it required to add maintainers that are listed in the [CNCF maintainer list](https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L178?)? Can we add maintainer email addresses who are not listed in that list?

Reply:
> is it required to add maintainers that are listed in the [CNCF maintainer list](https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L178?)?

No

> Can we add maintainer email addresses who are not listed in that list?

Yes